scripts/test_update_smoke.sh

```diff
@@ -146,17 +152,16 @@ fi
 
 # Create a 1.7.5 version Upgrade
 echo "---- Connecting to ${FORGE_CONNINFO} and running setup ----"
-$PSQL -c "DROP EXTENSION IF EXISTS timescaledb CASCADE" >>$LOGFILE 2>&1
-$PSQL -f pre.cleanup.sql >>$LOGFILE 2>&1
-$PSQL -c "CREATE EXTENSION timescaledb VERSION '${UPDATE_FROM_TAG}'" >>$LOGFILE 2>&1
+$PSQL -a -c "DROP EXTENSION IF EXISTS timescaledb CASCADE" >>$LOGFILE 2>&1
```

Comment:
Maybe we should add `-a` to `PSQL`?

Reply:
of course; I was adding them 1-by-1 at first where it was needed because it messed up `missing_versions` at that time

added `-a` to `PSQL` instead and I've fixed `missing_versions`